### ISSUE TITLE
Image Element contains cross origin data.

### ISSUE THREAD

**Issue by sotoea** [KilledByAPixel/LittleJS#3]
I downloaded the base project and tried simply opening the index.html file to test if it would magically work out of the box :P but instead I got this issue. Without having to dive myself into the engine files, any ideas as to how to provide a quick and easy solution to this?

```
Uncaught DOMException: Failed to execute 'texImage2D' on 'WebGLRenderingContext': The image element contains cross-origin data, and may not be loaded.
    at glCreateTexture (file:///D:/Node/LittleJS-main/engine/engineWebGL.js?26:151:15)
    at glInit (file:///D:/Node/LittleJS-main/engine/engineWebGL.js?26:23:21)
    at Image.tileImage.onload (file:///D:/Node/LittleJS-main/engine/engine.js?26:55:9)
```

Thanks in advance!

**Comment by KilledByAPixel** [comment]
I'm not sure what you are using to edit in but you need small web server to load files like images. This is a security measure that has been added to most modern browsers.

The easiest way is to use an editor like brackets which will do that for you...

https://brackets.io/

What I use is the Live Sever extension for visual studio code, which works real nice.

https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer

**Comment by sotoea** [comment]
For now I managed to fix it with a simple solution but I still wouldn't be able to run it without this extra step, gotta love cross origin crap.

Using nodejs http-server, I just started a server in my local project, now I can run it in that server and it works fine.

As I am writing this, I see @KilledByAPixel already provided a response, which is in line with what I mention here, thanks for the very quick reply! I highly appreciate it :) I can't wait to get something going with LittleJs.

Great work!
